stock_picking_invoice_link/models/sale_order.py

```diff
@@ -14,7 +14,8 @@ def get_stock_moves_link_invoice(self):
             lambda mv: (
                 mv.state == "done"
                 and not (
-                    any(
+                    not self.env.context.get("skip_check_invoice_state", False)
```

Comment:
@tafaRU I'm wondering if this cannot be included in function signature as it's better to understand ?

Reply:
Thanks for the suggestion but for my purpose (to use it in the script below) I'd prefer to keep it that way.

```
for inv_line in invoices.invoice_line_ids:
    product = inv_line.product_id
    if product.tracking == 'serial':
        for so_line in inv_line.sale_line_ids.with_context(skip_check_invoice_state=True):
            line_vals = so_line._prepare_invoice_line()
            move_line_ids = (
                    "move_line_ids" in line_vals and line_vals["move_line_ids"] or []
            )
            inv_line.write({"move_line_ids": move_line_ids})
```

Reply:
@rousseldenis did you see my answer?

Reply:
Ok see

Reply:
@tafaRU Can you put context value in a variable externally to lambda in order to avoid calling it each time ?

Reply:
@rousseldenis done